CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ if (POLICY CMP0063)
 endif (POLICY CMP0063)
 
 project(glog VERSION 0.5.0 LANGUAGES CXX)
-
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
Please remove. Glog still uses C++98/03. C++11 features and beyond are optional.